### PR TITLE
fix warnning sidebar propTypes &  update react-svg-pan-zoom to 3 by following the documents &  fix some type error

### PR DESCRIPTION
**Pull request by yangxyo**
Sidebar's width + height are not required any more.

### CODE REVIEW

**Comment by yangxyo** [comment]
update react-svg-pan-zoom to 3 by following the documents, and fix some type error

File: src/components/viewer2d/viewer2d.jsx [hunk 1]
```diff
@@ -353,16 +353,16 @@ export default function Viewer2D(
...
         onChangeValue={onChangeValue}
         tool={mode2Tool(mode)}
         onChangeTool={onChangeTool}
         detectAutoPan={mode2DetectAutopan(mode)}
         onMouseDown={onMouseDown}
         onMouseMove={onMouseMove}
         onMouseUp={onMouseUp}
-        miniaturePosition="none"
-        toolbarPosition="none"
+        toolbarProps={{position:"none"}}
```
Comment (MeTaNoV) [inline src/components/viewer2d/viewer2d.jsx:364]: Got this error with those value:

```
react.development.js:209 Warning: Failed prop type: Invalid prop `position` of value `none` supplied to `Ce`, expected one of ["top","right","bottom","left"].
    at Ce (webpack-internal:///../node_modules/react-svg-pan-zoom/build-umd/react-svg-pan-zoom.min.js:1:25279)
    at t (webpack-internal:///../node_modules/react-svg-pan-zoom/build-umd/react-svg-pan-zoom.min.js:1:31562)
    at div
    at Viewer2D (webpack-internal:///../src/components/viewer2d/viewer2d.jsx:106:20)
    at Content (webpack-internal:///../src/components/content.jsx:24:20)
    at div
    at ReactPlannerContent (webpack-internal:///../src/react-planner.jsx:51:21)
    at ReactPlanner (webpack-internal:///../src/react-planner.jsx:123:21)
    at ConnectFunction (webpack-internal:///../node_modules/react-redux/es/components/connectAdvanced.js:234:68)
    at div
    at Provider (webpack-internal:///../node_modules/react-redux/es/components/Provider.js:19:20)
    at Wrapper (webpack-internal:///./src/wrapper.jsx:73:64)
    ```

**Review by MeTaNoV — CHANGES_REQUESTED** [review]

**Comment by yangxyo** [comment]
1. add objExporter, save project with JSON and OBJ, 
2. fix new project then add woods in catalog will cras

File: src/components/viewer2d/viewer2d.jsx (hunk 1 above)
Comment (yangxyo) [reply src/components/viewer2d/viewer2d.jsx:364]: image

Did you upgrade ?
```
"react-svg-pan-zoom": "^3.12.1",
```

Comment (MeTaNoV) [reply src/components/viewer2d/viewer2d.jsx:364]: sorry, forgot to update indeed... Thanks again for the contribution!

**Review by MeTaNoV — APPROVED** [review]

**Pull request merged**